restapi/pom.xml

```diff
@@ -150,6 +150,16 @@
       <groupId>org.assertj</groupId>
       <artifactId>assertj-core</artifactId>
       <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>io.reactivex.rxjava2</groupId>
+      <artifactId>rxjava</artifactId>
+      <version>2.2.15</version>
```

Comment:
**IMPORTANT**: @dimas-b let's go straight for the RxJava 3 and RxJava `rxjava3-extensions`.. As I understood it requires only new package re-import. There is bunch of new stuff and way better support for Java 8. Please, let's start with latest, 2.x is in maintenance mode.

Reply:
Sure. I'll give it a try.

Reply:
Everything I can find shows `take(n)` as a synonym for `limit(n)`.